package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "express": "^4.16.4",
     "axios": "0.18.0",
+    "dotenv": "^6.2.0",
+    "express": "^4.16.4",
```

Comment:
Why do we have express installed on the frontend? Is there an instance we used it?

Reply:
This was employed in deploying the base app to Heroku.